### ISSUE TITLE
Using docker volume create with an unknown driver hangs

### ISSUE THREAD

**Issue by thaJeztah** [docker/docker#23452]
Using `docker volume create` with an unknown driver hangs, and doesn't give any feedback to the user;

```bash
$ docker volume create --driver blabla --name something
(just hangs)
```

Daemon logs show that *something* is happening;
```
WARN[0867] Unable to locate plugin: blabla, retrying in 1s
WARN[0868] Unable to locate plugin: blabla, retrying in 2s
```

I think we should improve this, possibly showing a message, such as:

```bash
$ docker volume create --driver blabla --name something
Error: driver "blabla" not found
```

Or, if we want to wait for it to become available;

```bash
$ docker volume create --driver blabla --name something
waiting for driver "blabla" to become active (press CTRL+c to cancel)
........ (dots show progress)
```

**Comment by HackToday** [comment]
For docker 1.9 or not sure if behavior changed recently, if you kick volume create with not available plugin, it would have following as:

```
Error looking up volume plugin ***: plugin not found
```

**Comment by yongtang** [comment]
The default timeout for plugin probe is 30s:

https://github.com/docker/docker/blob/master/pkg/plugins/plugins.go#L160-L165
https://github.com/docker/docker/blob/v1.11.2/pkg/plugins/client.go#L17-L21

After that it will error out.

In log:

```
... level=warning msg="Unable to locate plugin: blabla, retrying in 1s"
... level=warning msg="Unable to locate plugin: blabla, retrying in 2s"
... level=warning msg="Unable to locate plugin: blabla, retrying in 4s"
... level=warning msg="Unable to locate plugin: blabla, retrying in 8s"
... level=error msg="Handler for POST /v1.23/volumes/create returned error: create something: create something: Error looking u

```

I think copying the log warning messages to the stdout should be good to give user an indication of what is happening in the 30s.

**Comment by runcom** [comment]
the same happens with authorization plugins for the matter - eventually after 30s you get an error

**Comment by thaJeztah** [comment]
Yes, we should have at least *something* to indicate it's not stuck, but retrying. Wondering if copying the full log message would be too verbose, but guess it would work

@yongtang are you interested in opening a PR for that?

**Comment by yongtang** [comment]
@thaJeztah Sure let me take a look and create a pull request for it. #dibs

**Comment by ket4yii** [comment]
Any news about this issue? I have the same thing when I start docker daemon and the daemon hangs for, maybe, 10 minutes.

**Comment by matiwinnetou** [comment]
+1 this issue hit me as well

**Comment by cpuguy83** [comment]
This is not something we can address until we stop supporting V1 plugins.
It's not hanging, it's retrying to plugin with a backoff.

**Comment by aep** [comment]
how do you remove such a volume once it's created? it doesnt show up in docker volume ls
